ava-test/echo.js

```diff
@@ -129,3 +129,9 @@ test.cb('piping to a file with -n', t => {
     t.end();
   });
 });
+
+test('stderr with unrecognized options is null', t => {
```

Comment:
s/null/empty/ or s/null/falsy/

We may want to switch stderr from `null` to the empty string in a future release.